fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.dev.codacy.org/gh/devcodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=36026)
```suggestion
(function ($, window) {
```